### PR TITLE
Add postgres env

### PR DESCRIPTION
**Pull request by konrad-sol**
## Description
Add docker-compose, dash config and CI job definition with postgres.
sql job can be executed on demand and by push to branch with *postgres* keyword.

## Checklist

- [x] Make sure that tests are passing in githubaction
- [x] Check if change should be populated to other branches (master, release-3-lts, release-4.0)
- [x] Tests are stable

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**UI tests result: success** :white_check_mark: 
Branch used: refs/pull/175/merge
Commit:  
Triggered by: pull_request (@konrad-sol)
[Execution page](https://github.com/TykTechnologies/tyk-analytics-tests/actions/runs/6048166676)

**Comment by Tyk-ITS** [comment]
**UI tests result: success** :white_check_mark: 
Branch used: refs/heads/add_postgres_env
Commit: 892af1d439d211b334da04532cc464b405680035 fix in CI job
Triggered by: push (@konrad-sol)
[Execution page](https://github.com/TykTechnologies/tyk-analytics-tests/actions/runs/6048107408)

**Comment by Tyk-ITS** [comment]
**UI tests result: success** :white_check_mark: 
Branch used: refs/heads/add_postgres_env
Commit: d8cabb9580808b0c886b27646d09fdc8bd34df01 add "table_sharding": true in postgres dash config
Triggered by: push (@konrad-sol)
[Execution page](https://github.com/TykTechnologies/tyk-analytics-tests/actions/runs/6048596154)

**Comment by Tyk-ITS** [comment]
**UI tests result: success** :white_check_mark: 
Branch used: refs/pull/175/merge
Commit: d8cabb9580808b0c886b27646d09fdc8bd34df01 
Triggered by: pull_request (@konrad-sol)
[Execution page](https://github.com/TykTechnologies/tyk-analytics-tests/actions/runs/6048596342)

File: ci/conf/tyk_analytics_postgres.conf [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    "tyk_api_config": {
+        "Host": "http://tyk-gateway",
+        "Port": "8080",
+        "Secret": "352d20ee67be67f6340b4c0605b044b7"
+    },
+    "enable_ownership": true,
+    "storage": {
+        "main": {
+            "type": "postgres",
+            "connection_string": "user=tyk password=tyk database=tyk host=tyk-postgres port=5432"
```
[the remark this answers is not in the document]
Comment (konrad-sol) [reply ci/conf/tyk_analytics_postgres.conf]: added

**Comment by Tyk-ITS** [comment]
**UI tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: f55c2f8884fa1484d83fdafb4d7ddd0364b20196 add postgres ens
Triggered by: push (@konrad-sol)
[Execution page](https://github.com/TykTechnologies/tyk-analytics-tests/actions/runs/6049062395)

**Pull request merged**